src/api/search.ts

```diff
@@ -4,8 +4,6 @@ import Office from '../types/Office'
 import SearchMeta from '../types/SearchMeta'
 import apiClient from './client'
 
-export type SearchResult = Agent | Office
```

Comment:
Is this removal because the type is not used elsewhere and there's a component by the same name?

Reply:
Yes and because SearchData may now contain many types of results, not just Agent and office